package-lock.json

```diff
@@ -1185,7 +1185,7 @@
     "arr-diff": {
       "version": "4.0.0",
       "resolved": "https://registry.npmjs.org/arr-diff/-/arr-diff-4.0.0.tgz",
-      "integrity": "sha512-YVIQ82gZPGBebQV/a8dar4AitzCQs0jjXwMPZllpXMaGjXPYVUawSxQrRsjhjupyVxEvbHgUmIhKVlND+j02kA==",
+      "integrity": "sha1-1kYQdP6/7HHn4VI1dhoyml3HxSA=",
```

Comment:
same integrity issue as before, try the steps and see if they work for not updating the integrity:

```
rm -rf node_modules
npm cache clear --force
rm package-lock.json
npm install
```

hopefully should be no diffs in the integrity after this? Should all be 512s